web/client/components/geostory/layouts/sections/Immersive.jsx

```diff
@@ -32,9 +33,13 @@ const Immersive = ({
     sectionType,
     inViewRef,
     viewWidth,
-    viewHeight
-}) => (
-    <section
+    viewHeight,
+    focusedContent,
+    contentId
+}) => {
+    const hideTitle = focusedContent && focusedContent.isBackground && (get(focusedContent, "target.id") === contentId);
```

Comment:
hideContents, can be an option of focusedContent instead of calling it isBackground